docs/v3/core/v3.0.rst

```diff
@@ -634,13 +634,12 @@ mandatory names:
 ``codecs``
 ^^^^^^^^^^
 
-    Specifies a list of codecs to be used for encoding and decoding chunks. The
-    value must be an array of objects, each object containing a member with
-    ``name`` whose value is a string referring to a v3 codec specification. The
-    codec object may also contain a ``configuration`` object which consists of
-    the parameter names and values as defined by the corresponding codec
-    specification.  Since an ``array -> bytes`` codec must be specified, the
-    list cannot be empty.
+    Specifies a list of codecs to be used for encoding and decoding chunks.  The
+    value MUST be an array of objects, where each object contains a member with the name
+    ``name`` whose value is a string. The ``name`` member identifies the specification for the codec.  A
+    codec object MAY contain a member named ``configuration``, which is an object defined by the respective codec
```

Comment:
I think this should be changed back to "parameter names and values" instead of "object", since this is a) out of scope of this PR, and b) I think parameter names and values makes more sense because this section is taking about a JSON document (I think??).